core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1712,7 +1714,7 @@ class KafkaConfig private(doLog: Boolean, val props: java.util.Map[_, _], dynami
   def minInSyncReplicas = getInt(KafkaConfig.MinInSyncReplicasProp)
   def logPreAllocateEnable: java.lang.Boolean = getBoolean(KafkaConfig.LogPreAllocateProp)
 
-  // We keep the user-provided String as `ApiVersion.apply` can choose a slightly different version (eg if `0.10.0`
+  // We keep the user-provided String as `MetadataVersion.apply` can choose a slightly different version (eg if `0.10.0`
```

Comment:
We don't have an `apply` method anymore. Need to reference the new fromVersionString we added